OpenShift-Installation/Troubleshooting-Guide.rst

```diff
@@ -520,3 +520,145 @@ After the OpenShift nodes come up after the Ansible installation, perform the fo
              *A:Dut-H# configure vswitch-controller shutdown 
              *A:Dut-H# configure vswitch-controller no shutdown
 
+Uninstall a Nuage Openshift Cluster
+===================================================
+
+To uninstall the Nuage Openshift Cluster, delete the Nuage components manually and then uninstall Openshift using the ansible playbook.
+
+.. Note:: It is mandatory to delete all user created zones, subnets, policies etc on the Openshift Cluster and VSD manually before uninstalling the Openshift cluster. This is to ensure consistency between Nuage VSD & the Openshift cluster data.
```

Comment:
I'd make this: 
`.. Note:: It is mandatory to delete all user created zones, subnets, policies etc on the Openshift Cluster and the matching VSD Domain manually before uninstalling the Openshift cluster. This is to ensure consistency between Nuage VSD & the Openshift cluster data.`

You don't need to delete the whole VSD ;)